packages/toolkit/src/lib/react-query-service/knowledge/useGetFileDetails.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from "@tanstack/react-query";
+import { getInstillAPIClient, Nullable } from "@instill-ai/toolkit";
+
+export function useGetFileDetails() {
+  return useQuery({
+    queryKey: ["fileDetails"],
+    queryFn: async ({ ownerId, kbId, fileUid, accessToken }: { ownerId: string, kbId: string, fileUid: string, accessToken: Nullable<string> }) => {
+      if (!accessToken) {
+        return Promise.reject(new Error("accessToken not provided"));
+      }
+
+      const client = getInstillAPIClient({ accessToken });
+
+      const response = await client.vdp.artifact.getFileDetails({ ownerId, kbId, fileUid });
+
+      return Promise.resolve(response.file);
+    },
+  });
+}
```

Comment:
This should not be written like this way

The parameter should be expose at the outside of queryFn

Please check how we write the query function for the pipeline, all the query function has this issue. We need to unify it